contracts/vesting/src/contract.rs

```diff
@@ -144,14 +147,18 @@ pub fn execute(
 ///
 /// Callable by ADMIN only, see [instantiate].
 pub fn try_update_locked_pledge_cap(
-    amount: Uint128,
+    address: String,
+    cap: PledgeCap,
     info: MessageInfo,
     deps: DepsMut,
 ) -> Result<Response, ContractError> {
     if info.sender != ADMIN.load(deps.storage)? {
         return Err(ContractError::NotAdmin(info.sender.as_str().to_string()));
     }
-    update_locked_pledge_cap(amount, deps.storage)?;
+    let mut account = account_from_address(&address, deps.storage, deps.api)?;
+
+    account.pledge_cap = Some(cap);
+    // update_locked_pledge_cap(amount, deps.storage)?;
```

Comment:
Again, this is not specific to this PR but to the code that's getting merged, so question to @durch : why is this call commented? Don't we need it?

Reply:
Fixed on the release branch, and I've cherry-picked that commit ontop of this branch